packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -225,7 +226,7 @@ describe('Ceramic anchoring', () => {
     ])
     const controller = ceramic1.context.did.id
 
-    const doctype1 = await ceramic1.createDocument(DOCTYPE_TILE, { content: { x: 1 } }, { anchor: false, publish: false })
+    const doctype1 = await ceramic1.createDocument<TileDoctype>(DOCTYPE_TILE, { content: { x: 1 } }, { anchor: false, publish: false })
```

Comment:
Kind of annoying that you have to both specialize on the doctype subclass AND provide the doctype name string as an arg.  Maybe in the future we could remove the string doctype name argument and get it from the type specialization.  Mostly just an observation, no need to do anything on that here